ocs_ci/utility/rhv.py

```diff
@@ -409,3 +410,77 @@ def remove_disks(self, vm, detach_only=True):
                 self.remove_disk(
                     vm=vm, identifier=disk.id, key="id", detach_only=detach_only
                 )
+
+    def get_vm_status(self, vm_name):
+        """
+        Get the power status of RHV VM
+
+        Args:
+           vm_name (str): RHV VM name
+
+        Returns :
+           str: Power status of RHV VM
+
+        """
+        vm = self.get_vms_by_pattern(pattern=vm_name)[0]
+        vm_service = self.vms_service.vm_service(vm.id)
+        vm_info = vm_service.get()
+        return vm_info.status
+
+    def stop_rhv_vms(self, vm_names, force=False):
+        """
+         Shutdown the RHV virtual machines
+
+        Args:
+            vm_names (list): Names of RHV vms
+            force (bool): True for non-graceful VM shutdown, False for
+                graceful VM shutdown.
+
+        """
+        for vm_name in vm_names:
+            # Find the virtual machine
+            vm = self.get_vms_by_pattern(pattern=vm_name)[0]
```

Comment:
To be consistent with existing functions such as `add_disk` or `remove_disk`, we could get the Vm instance in the `platform_nodes.py` itself and then directly pass the Vm instance in `rhv.py` functions.